pg_backup_api/pg_backup_api/utils.py

```diff
@@ -16,39 +16,59 @@
 # You should have received a copy of the GNU General Public License
 # along with Postgres Backup API.  If not, see <http://www.gnu.org/licenses/>.
 
+"""
+Utilitary functions and constants used through pg-backup-api code.
+
+:var API_CONFIG: configuration of pg-backup-api. The ``supported_options`` key
+    contains a tuple of supported options for ``barman recover`` command.
+:var CONFIG_FILENAME: path to the main Barman configuration file.
+:var LOG_FILENAME: path to the file where pg-backup-api logs its messages.
+"""
 from logging.config import dictConfig
+from typing import Optional, TYPE_CHECKING
 
 from flask import Flask
 
 import barman
 from barman import config
 from barman.infofile import BackupInfo
 
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
On python 3.6 and 3.7 at least (I haven't tested other versions yet) the imports on the following lines are still required, even if `TYPE_CHECKING` is `False`. Currently it fails as follows:

```
$ ~/.local/bin/pg-backup-api 
Traceback (most recent call last):
  File "/var/lib/barman/.local/bin/pg-backup-api", line 11, in <module>
    load_entry_point('pg-backup-api', 'console_scripts', 'pg-backup-api')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 476, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2700, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2318, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/pg-backup-api/pg_backup_api/pg_backup_api/__init__.py", line 20, in <module>
    import pg_backup_api.logic.utility_controller
  File "/pg-backup-api/pg_backup_api/pg_backup_api/logic/utility_controller.py", line 31, in <module>
    from pg_backup_api.utils import load_barman_config, get_server_by_name, parse_backup_id
  File "/pg-backup-api/pg_backup_api/pg_backup_api/utils.py", line 46, in <module>
    def create_app() -> flask.app.Flask:
NameError: name 'flask' is not defined
```

Reply:
Oh, I missed quoting these.